clean_old_aggregates.py

```diff
@@ -0,0 +1,70 @@
+import keystoneauth1
+import datetime
+from keystoneauth1.identity import v3
+from keystoneauth1 import loading
+from keystoneauth1 import session
+from novaclient import client as nova_client
+from blazarclient import client as blazar_client
+
+def main():
+
+    loader = loading.get_plugin_loader('password')
```

Comment:
These `loading` functions are intended to allow creating an Auth/Session from either a set of oslo `Opt` types, or flags from the CLI. You should instead invoke the `v3.Password` constructor to create an object that represents "user is authentication with user and password".

But... you don't have to deal with any of this. You can use the `osapi` and `osrest` helper modules already existing in the `hammers` repo.

```python
import sys

from hammers import osapi, osrest

# Let's parse some CLI arguments
parser = argparse.ArgumentParser(description='Clean old Nova aggregates tied to expired Blazar leases.')
# The 'osapi' module has a helper to add rules to parse well-known arguments
# (like the OpenStack username/password)
osapi.add_arguments(parser)
# Invoke the parser on the CLI arguments, save as 'args'
args = parser.parse_args(sys.argv[1:])
# Pass args to Auth constructor; will fall back to finding values in well-known env vars
# (like OS_USERNAME/OS_PASSWORD)
auth = osapi.Auth.from_env_or_args(args=args)

# Get aggregates...
aggregates = osrest.nova.aggregates(auth)